aip/general/0136.md

```diff
@@ -174,8 +173,17 @@ RPC name itself causes confusion, and can even cause issues for client libraries
 which generate both synchronous and asynchronous methods to call the RPC in some
 languages.
 
+### Choice of HTTP methods for stateless methods
+
+Stateless method **should** use `GET` HTTP method for retrieval use cases due to
+its cache-friendly nature. This can reduce the costs for both the server and the
```

Comment:
it might be worth elaborating: e.g. it's common for web browsers to cache GET requests, and it's understood by clients to be a cacheable response.

Reply:
Thanks for the review! Can you elaborate a bit more on "and it's understood by clients to be a cacheable response" part? What does this mean? Specifically what is "it" referring to? Thanks!

Reply:
Sorry, by "it" I was referring to the fact that GET methods are cacheable. 

> and it's understood by clients to be a cacheable response

What I mean is HTTP clients often cache GET responses as GET implies that it's doing data retrieval: https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.1. Specficially:

> The response to a GET request is cacheable; a cache MAY use it to
   satisfy subsequent GET and HEAD requests unless otherwise indicated
   by the Cache-Control header field ([Section 5.2 of [RFC7234]](https://datatracker.ietf.org/doc/html/rfc7234#section-5.2)).

So custom methods which are cacheable and send no payload should use GET.

Reply:
Thanks, I edited it again with a new commit. PTAL!